lib/tasks/migrations.rake

```diff
@@ -9,8 +9,14 @@ namespace :migrations do
     require File.join(File.dirname(__FILE__), '..', '..', 'config', 'environment')
     puts AppConfig.environment.s3.key
 
-    connection = Aws::S3.new( AppConfig.environment.s3.key, AppConfig.environment.s3.secret)
-    bucket = connection.bucket(AppConfig.environment.s3.bucket)
+    connection = Fog::Storage.new({
+      provider:              AppConfig.environment.s3.provider.get,
```

Comment:
Layout/FirstHashElementIndentation: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.